### PR TITLE
Added search loader

### LINKED ISSUE #15616: Replace rotating magnifier icon with the spinner component
### Problem to solve

When redesigning the dashboard, we added for fun a rotating magnifier icon as a search loading indicator. This issue is a placeholder for improving the search UX.

https://user-images.githubusercontent.com/120486/211091661-d50687ff-fea0-4332-a5be-c57a4dcad797.mov

### Proposal

A good MVC could be to replace the magnifier icon with the spinner component while performing the search.

### PR DESCRIPTION
**Pull request by Ciggzy1312**
Signed-off-by: Deepayan Mukherjee <deepayanmukherjee1312@gmail.com>

## Description
Removed the rotating magnifier icon for loading and added a spinner animation

## Related Issue(s)

Fixes #15616 

## Release Notes

```release-note
NONE
```

## Build Options:

- [ ] /werft with-github-actions
      Experimental feature to run the build with GitHub Actions (and not in Werft).
- [ ] /werft with-local-preview
      If enabled this will build `install/preview`
- [x] /werft with-preview
- [ ] /werft with-large-vm
- [ ] /werft with-integration-tests=all
      Valid options are `all`, `workspace`, `webapp`, `ide`, `jetbrains`, `vscode`, `ssh`

### CODE REVIEW

**Comment by roboquat** [comment]
@Ciggzy1312: Adding the "do-not-merge/release-note-label-needed" label because no release-note block was detected, please follow our [release note process](https://git.k8s.io/community/contributors/guide/release-notes.md) to remove it.

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

**Comment by andrew-farries** [comment]
https://user-images.githubusercontent.com/8225907/213435210-42e90668-b8a1-4b32-a7a1-13b85c82b52f.mov

**Review by andrew-farries — APPROVED** [review]

**Review by gtsiolis — APPROVED** [review]
Thanks for contributing, @Ciggzy1312! 🍊  

✔️  UX LGTM, from what I see in the screencast that @andrew-farries posted in https://github.com/gitpod-io/gitpod/pull/15804#issuecomment-1396852979.

❓  However, do you think we could reuse the [`Loader`](https://github.com/gitpod-io/gitpod/blob/e940a97c0fe5ab3709a37918a71d07b26a8917a9/components/dashboard/src/components/Loader.tsx) component instead of repeating the SVGs? This could be fine as a next iteration.

❗  You'll also need to sign a Contributor License Agreement (CLA) form[[1](https://powerforms.docusign.net/30e1aa46-cce5-4bc6-a933-ad02cd8d5ada?env=eu&acct=86335c2c-01b9-4edc-a187-014f5150a112&accountId=86335c2c-01b9-4edc-a187-014f5150a112)] once before merging your first contribution. Cc [@meysholdt](https://github.com/meysholdt)

/hold

**Comment by Ciggzy1312** [comment]
Thanks @gtsiolis 
Yeah I will take up reusing the Loader component in a separate PR if that sounds okay
And I did fill in the CLA form

**Comment by meysholdt** [comment]
Thank you @Ciggzy1312 for signing the CLA!

**Pull request merged**
